lib/diagnostic.js

```diff
@@ -32,7 +41,16 @@ db.dropDatabase();
 // Jon Arryn
 
 // your code begins here
+db.houses.update({name: "House Stark"}, {$set: {members: "Ned Stark"} });
```

Comment:
These calls to `update` will keep overwriting the value of `member` with a string.  You were also supposed to use `$push` at least once.

```javascript
db.houses.update({
  name: 'House Stark'
}, {
  $set: {
    members: ['Ned Stark']
  }
});

db.houses.update({
  name: 'House Targaryen'
}, {
  $set: {
    members: ['Viserys Targaryen']
  }
});

db.houses.update({
  name: 'House Arryn'
}, {
  $set: {
    members: ['Jon Arryn']
  }
});

db.houses.update({
  name: 'House Stark'
}, {
  $push: {
    members: {
      $each: ['Arysa Stark', 'Sansa Stark']
    }
  }
});

db.houses.update({
  name: 'House Targaryen'
}, {
  $push: {
    members: 'Daenerys Targaryen'
  }
});
```